plugin/trino-delta-lake/src/test/resources/trino439/cdc_table/README.md

```diff
@@ -0,0 +1,5 @@
+In a test:
+```java
+getQueryRunner().execute("CREATE TABLE cdc_table WITH (partitioned_by=ARRAY['key'], change_data_feed_enabled=true, location = 'file:/<path>/plugin/trino-delta-lake/src/test/resources/trino439/cdc_table') AS SELECT 1 key, 'a' value");
```

Comment:
Could you pick a better name for the directory than `trino439` ? The release number isn't relevant here
Maybe just `cdc_table` ?

Reply:
It is consistent with the naming of the sibling directories. We could use just trino/cdc_table?

Reply:
let's use `trino/cdc_table`